macos/FluentUI/Button/Button.swift

```diff
@@ -174,6 +176,7 @@ open class Button: NSButton {
 		}
 	}
 	
+	/// Tint color of the button content.
 	open override var contentTintColor: NSColor? {
 		get {
 			if style == .primaryFilled {
```

Comment:
Note: This should probably also be @obj open override, and I probably just forgot to do so

Reply:
This would be the main thing, otherwise LGTM!

Reply:
I made it public since we don't want it to be overridable.